scipy/signal/_ltisys.py

```diff
@@ -3447,7 +3447,7 @@ def dbode(system, w=None, n=100):
     Returns
     -------
     w : 1D ndarray
-        Frequency array [rad/time_unit]
+        Frequency array [rad/s]
```

Comment:
Avoiding the unit `rad` is more beginner friendly. What about:
```text
Angular frequency in `2*pi/sampling interval`
```
A personal knit pick: Though using brackets to denote units  (like `[rad/s]`) is widely used, it is not officially recommend, as can be seen in the [SI Brochure].

[SI Brochure]: https://www.bipm.org/documents/20126/41483022/SI-Brochure-9.pdf

Reply:
I don't think avoiding `rad` is helpful... Those *are* radians per unit of time, that's the definition of angular frequency. One can't be using these functions without being comfortable with such definitions.

The sampling interval isn't relevant here, either.

The *unit* of the sampling interval of the system is what's relevant, and other functions document it as seconds, that's why I changed this to use seconds.

Reply:
Wouldn't adding a little additional explanation be helpful nonetheless? E.g.:
```
Array of frequencies normalized to the Nyquist frequency being ``np.pi/dt`` 
with ``dt`` being the sampling interval of the `system` parameter.  The unit is rad/s 
assuming ``dt`` is in seconds.

Reply:
> However, all functions which take a dt document it as seconds...

That's a mistake that should be corrected.

Reply:
There seems to be consensus on removing the assumption of seconds.

I will try and include a commit for that in this PR, is that ok?

Reply:
Great! If you want, you could also beautify the graph a bit, as mentioned [here](https://github.com/scipy/scipy/pull/20908#pullrequestreview-2228572028) (though it is not critical).